src/widgetastic/xpath.py

```diff
@@ -5,6 +5,8 @@
 
 def quote(s):
     """Quotes a string in such a way that it is usable inside XPath expressions."""
+    if not s:
```

Comment:
should this act different for a empty string?

Reply:
```suggestion
    if s is None:
```

Reply:
empty string should be quoted as empty string, but explicit comparison is clearer